tests/e2e/linalg_ext_ops/CMakeLists.txt

```diff
@@ -74,6 +75,7 @@ iree_check_single_backend_test_suite(
   NAME
     check_llvm-cpu_local-task
   SRCS
+    "attention.mlir"
```

Comment:
This test is failing on emscripten with `--iree-llvm-target-triple=wasm32-unknown-emscripten` (tested on post-submit only, best-effort to maintain): https://github.com/iree-org/iree/actions/runs/4026898563/jobs/6922078074

```
FAILED: tests/e2e/linalg_ext_ops/check_llvm-cpu_local-task_attention.mlir_module.vmfb /work/build-emscripten/tests/e2e/linalg_ext_ops/check_llvm-cpu_local-task_attention.mlir_module.vmfb 
cd /work/build-emscripten/tests/e2e/linalg_ext_ops && /work/full-build-dir/install/bin/iree-compile --output-format=vm-bytecode --mlir-print-op-on-diagnostic=false --iree-hal-target-backends=llvm-cpu /work/tests/e2e/linalg_ext_ops/attention.mlir -o check_llvm-cpu_local-task_attention.mlir_module.vmfb --iree-hal-executable-object-search-path=\"/work/build-emscripten\" --iree-llvm-target-triple=wasm32-unknown-emscripten
/work/tests/e2e/linalg_ext_ops/attention.mlir:6:8: error: 'memref.alloca' op all stack allocations need to be hoisted to the entry block of the function
  %1 = iree_linalg_ext.attention ins(%query, %key, %value : tensor<1x4x4xf32>,
       ^
```

Does that error make sense to you?

Reply:
Is this happening only in the wasm backend... or is it on all backends?

Reply:
Just the wasm compilation (which is just the CPU path but using `--iree-llvm-target-triple=wasm32-unknown-emscripten`). We might see similar errors with other target triples or LLVM/codegen options, but all of our other CI builds and tests are passing.

Reply:
Surprising that this is happening on wasm, and not outside of it... This should be using the CPU compilation pipeline directly right? If so, then I can look into it.

Reply:
https://github.com/iree-org/iree/pull/11992 fixes this.